### PR TITLE
feat(traces): More improvements, use sdk names now

### PR DESCRIPTION
**Pull request by k-fish**
### Summary
This uses sdk names as well as project to differentiate when it comes to multi-service single project traces. This also adds some small UI tweaks.

### CODE REVIEW

File: static/app/views/performance/traces/content.tsx [hunk 1]
```diff
@@ -399,6 +402,10 @@ function useTraces<F extends string>({
       sort,
       per_page: limit,
       maxSpansPerTrace: 10,
+      // breakdownCategory: ['db'],
+      breakdownCategory: [],
+      minBreakdownDuration: 2,
+      minBreakdownPercentage: 1 / 40,
```
Comment (Zylphrex) [inline static/app/views/performance/traces/content.tsx:408]: Did we want to keep the min breakdown duration and percentage now that we're not using the categories?

File: static/app/views/performance/traces/utils.tsx [hunk 2]
```diff
@@ -9,3 +10,28 @@ export function normalizeTraces(traces: TraceResult<string>[] | undefined) {
     (t1, t2) => (t1.name ? '0' : '1').localeCompare(t2.name ? '0' : '1')
   );
 }
+
+export function getStylingSliceName(
+  sliceName: string | null,
+  sliceSecondaryName: string | null
+) {
+  if (sliceSecondaryName) {
+    return sliceSecondaryName.slice(-2) + (sliceName ? sliceName.slice(0, 3) : '');
```
Comment (Zylphrex) [inline static/app/views/performance/traces/utils.tsx:19]: Could we leave a comment explaining why this is done this way? Without context, this is very cryptic.

File: static/app/views/performance/traces/utils.tsx [hunk 3]
```diff
@@ -9,3 +10,28 @@ export function normalizeTraces(traces: TraceResult<string>[] | undefined) {
...
+  }
+
+  return sliceName;
+}
+
+const PRIORITIZED_BREAKDOWN_OPS = ['db'];
```
Comment (Zylphrex) [inline static/app/views/performance/traces/utils.tsx:25]: Are we keeping the db stuff?

**Review by Zylphrex — APPROVED** [review]

File: static/app/views/performance/traces/content.tsx (hunk 1 above)
Comment (k-fish) [reply static/app/views/performance/traces/content.tsx:408]: Nah I'll remove em

File: static/app/views/performance/traces/utils.tsx (hunk 3 above)
Comment (k-fish) [reply static/app/views/performance/traces/utils.tsx]: nope

**Pull request merged**